src/store/sagas/movies.js

```diff
@@ -26,14 +27,25 @@ const getSortFilter = (filter) => {
 };
 
 function* loadMovies(action) {
-  const { page, filter } = action.payload;
+  const {
+    page,
+    filter,
+    year,
+    vote_average,
+    genre = '',
+    sortBy,
+  } = action.payload;
 
   try {
-    const { movies, totalPages } = yield call(moviesApi.get, {
+    const { movies, totalPages } = yield call(moviesApi.get, queryString.stringify({
```

Comment:
It's not right, previous version was right. Please check out my comment about `paramSerializer` above